com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/UpsampleScene.compute

```diff
@@ -25,9 +25,9 @@ void MainUpsample (uint3 dti : SV_DispatchThreadID)
 
     float2 uvs = ((float2)dti.xy + 0.5f) * _ViewPortSize.zw;
 #if defined(UPSAMPLE_COLOR)
-    _OutputTexture[COORD_TEXTURE2D_X(dti.xy)] = Bilinear(_InputTexture, uvs);
+    _OutputTexture[COORD_TEXTURE2D_X(dti.xy)] = Sample(TEXTURE2D_X_ARGS(_InputTexture, s_linear_clamp_sampler), uvs);
 #endif
```

Comment:
I think you removed this pass altogether with TAAU, since now we are just doing a min/max upsampler on depth and a trilinear on motion vectors.
Thus, I would say we remove this shader altogether as well.

Reply:
Indeed, I even removed it C# side, just forgot the shader side. Will do now.

Reply:
Done